apps/andi/lib/andi_web/live/ingestion_live_view/metadata/metadata_form.ex

```diff
@@ -109,4 +109,16 @@ defmodule AndiWeb.IngestionLiveView.MetadataForm do
       dataset -> dataset.business.dataTitle
     end
   end
+
+  defp get_dataset_names(ids) do
```

Comment:
do we need get_dataset_name if we have the names func?

Reply:
no we do not.